src/test/java/com/epam/tc/hw2/ex1/LoginPageTest.java

```diff
@@ -0,0 +1,120 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class LoginPageTest extends AbstractBeforeAfter {
+    WebDriverWait webDriverWait;
+    SoftAssertions softly = new SoftAssertions();
+    static final String EPAM_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @Test
+    public void LoginCheckTest() {
+        webDriverWait = new WebDriverWait(webDriver, 10);
+        webDriver.manage().window().maximize();
+        webDriver.navigate().to(EPAM_URL);
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        //Assert Browser title
+        webDriverWait.until(ExpectedConditions.titleIs("Home Page"));
+        softly.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+        //Assert performing login
+        WebElement loginForm = webDriver.findElement(By.id("user-icon"));
+        loginForm.click();
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(loginForm));
+        WebElement username = webDriver.findElement(By.id("name"));
+        username.sendKeys("Roman");
+        webDriver.findElement(By.id("password"));
+        WebElement password = webDriver.findElement(By.id("password"));
+        password.sendKeys("Jdi1234");
+        WebElement loginButton = webDriver.findElement(By.id("login-button"));
+        loginButton.click();
+        webDriverWait.until(ExpectedConditions.urlToBe(EPAM_URL));
+        softly.assertThat(webDriver.getCurrentUrl()).isEqualTo(EPAM_URL);
+        // Check loggined
+        WebElement userNameLogged = webDriver.findElement(By.id("user-name"));
+        userNameLogged.isDisplayed();
+        webDriverWait.until(ExpectedConditions.textToBePresentInElement(userNameLogged, "ROMAN IOVLEV"));
+        softly.assertThat(userNameLogged.getText()).isEqualTo("ROMAN IOVLEV");
+        //Assert that there are 4 items on the header section are displayed and they have proper texts
+        WebElement homeOnTheHeader = webDriver.findElement(By.xpath("//a[@href='index.html']"));
+        homeOnTheHeader.isDisplayed();
+        WebElement contactsOnTheHeader = webDriver.findElement(By.xpath("//a[@href='contacts.html']"));
+        contactsOnTheHeader.isDisplayed();
+        WebElement serviceOnTheHeader = webDriver.findElement(By.cssSelector(".dropdown"));
+        serviceOnTheHeader.isDisplayed();
+        WebElement metalsColors = webDriver.findElement(By.xpath("//a[@href='metals-colors.html']"));
+        metalsColors.isDisplayed();
+        softly.assertThat(homeOnTheHeader.getText()).isEqualTo("HOME");
+        softly.assertThat(contactsOnTheHeader.getText()).isEqualTo("CONTACT FORM");
+        softly.assertThat(serviceOnTheHeader.getText()).isEqualTo("SERVICE");
+        softly.assertThat(metalsColors.getText()).isEqualTo("METALS & COLORS");
```

Comment:
All web elements for the menu could be getter as `List<WebElement>` and assertion could be used through collection

Reply:
Looks like this comment was ignored